### DISCUSSION TITLE
Add AexOption for whether to preserve project folder structure

### DISCUSSION THREAD

**Discussion by zlovatt** [KraftyFX/aex#65] [Ideas]
Right now we _always_ serialize (and deserialize) the entire AE project folder hierarchy. (see: `estk/src/item/folder.tsx > getParentFolders()`)

There are cases where this isn't necessary, and you'd want items to just be imported into the project root.

Discussion: should this be a serialization option, a deserialization option, or both?

**Comment by rafikhan** [comment]
When you work with other (im|ex)porters what options do they give you for path processing? I'm guessing it's on import which is the equivalent of deserialization. With that I'm inclined to say just deserialization but am open to something else.

**Comment by zlovatt** [comment]
Good point. I think we should serialize the full structure and have this be a deserialization option only.
